app/views/users/edit.html.erb

```diff
@@ -74,27 +76,11 @@
     <% end %>
 
     <% if @user.errors.any? %>
-      <div id="error_explanation">
-        <h2><%= pluralize(@user.errors.count, "error") %> prohibited your profile from being saved:</h2>
-
-        <ul>
-          <% @user.errors.full_messages.each do |message| %>
-            <li><%= message %></li>
-          <% end %>
-        </ul>
-      </div>
+      <%= render partial: "users/errors", locals: { errors: @user.errors, title_suffix: "prohibited your profile from being saved" } %>
```

Comment:
prohibited is such a strong word 😅 I feel like prevented should be used here but that's outside the scope of this PR.